etl/chart_revision/cli.py

```diff
@@ -12,6 +12,8 @@
 from rich_click.rich_command import RichCommand
 from structlog import get_logger
 
+# TBD
```

Comment:
TBD means "To be determined", I guess you could specify "To be deleted after testing period".